components/device-mgt/org.wso2.carbon.device.mgt.core/src/test/resources/sql/h2.sql

```diff
@@ -75,6 +63,20 @@ CREATE TABLE IF NOT EXISTS DM_ENROLMENT (
     CONSTRAINT fk_dm_device_enrolment FOREIGN KEY (DEVICE_ID) REFERENCES
     DM_DEVICE (ID) ON DELETE NO ACTION ON UPDATE NO ACTION
 );
+
+CREATE TABLE IF NOT EXISTS DM_ENROLMENT_OPERATION_MAPPING (
```

Comment:
spelling mistake on DM_ENROLMENT_OPERATION_MAPPING, ENROLMENT should be ENROLLMENT